content/docs/2.15/_index.md

```diff
@@ -1,8 +1,20 @@
 +++
-title = "The KEDA Documentation"
+title = "Getting Started"
 weight = 1
 +++
 
-Welcome to the documentation for **KEDA**, the Kubernetes Event-driven Autoscaler. Use the navigation to the left to learn more about how to use KEDA and its components.  
+Welcome to the documentation for **KEDA**, the Kubernetes Event-driven Autoscaler. 
 
-Additions and contributions to these docs are managed on [the keda-docs GitHub repo](https://github.com/kedacore/keda-docs).
+Use the navigation bar on the left to learn more about KEDA's architecture  and how to deploy and use KEDA.
+
+Where to go
+===========
```

Comment:
Let's use the existing formatting, likely `#` or `##` depending on the header level you want